controllers/suite/clusters/humiocluster_controller_test.go

```diff
@@ -2150,11 +2274,6 @@ var _ = Describe("HumioCluster Controller", func() {
 			}
 			toCreate := suite.ConstructBasicSingleNodeHumioCluster(key, true)
 			toCreate.Spec.Image = oldSupportedHumioVersion
-			// ZOOKEEPER_URL gets filtered out by default in the call to ConstructBasicSingleNodeHumioCluster, so we add it back here
```

Comment:
This seems to no longer be true. While testing I found that there were duplicate "ZOOKEEPER_URL" environment variables until I removed this chunk.